addon/lib/helperworker.js

```diff
@@ -81,6 +84,20 @@ function resetPageMod(backend) {
         user.deleteEverything();
       }));
 
+      worker.port.on("contentTourComplete", watchFunction(function () {
+        const winutil = require("sdk/window/utils");
+        let win = winutil.getMostRecentBrowserWindow();
+        let magicCookie = "toggle-button--jid1-neeaf3sahdkhpajetpack-pageshot-recall";
+        let target = {
+          node: win.document.getElementById(magicCookie),
```

Comment:
I don't think this is e10s compatible?  Maybe it works via the cross-process object wrapper, but we should avoid writing code that uses CPOWs.  (This might be signalling a warning?)